galley/pkg/crd/validation/validation.go

```diff
@@ -98,14 +98,14 @@ func RunValidation(ready chan<- struct{}, stopCh chan struct{}, vc *WebhookParam
 	if err != nil || vc.Clientset == nil {
 		log.Fatalf("cannot create validation webhook service: %v", err)
 	}
-	if livenessProbeController != nil {
+	if vc.Mux == nil && livenessProbeController != nil {
```

Comment:
Yes, pilot has a readiness check already - and we want http-style check so it works with load balancers as well. 

Ideally Pilot will only expose port 15012 - it can handle GRPC and H2 on same port, with DNS certificate so can be used from any tool. And will also be able to authenticate with JWT tokens (for the SDS - we can use the same auth for other endpoints ). So a https probe will work too.

Note that with this PR we end up with 4 monitoring ports, 3 or 4 ctrlz ports - they are high priority for cleanup.